server/schema/catalog.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TYPE event AS ENUM ('guest speaker', 'study-trip', 'workshop');
+CREATE TYPE subject AS ENUM ('life skills', 'science', 'technology', 'engineering', 'math', 'college readiness');
+CREATE TYPE year AS ENUM ('junior', 'senior', 'both');
+
+DROP TABLE catalog IF EXISTS;
```

Comment:
y'all are really close! but one minor change. i'm pretty sure that it's `DROP TABLE IF EXISTS [table name]`. After you change that your all set

Reply:
fixed! mb